packages/unified-selection/src/unified-selection/iModel/IModel.ts

```diff
@@ -0,0 +1,136 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+/** @packageDocumentation
+ * @module UnifiedSelection
+ */
+
+/**
+ * Used as an argument to a function that can accept one or more [[Id64String]]s.
+ */
+export type IdArg = string | Set<string> | string[];
```

Comment:
I don't think we need it here, we can import `Id64Arg` from `core-bentley`

Reply:
Switched to using `Id64Arg`.